pkg/awsManager/ec2Manager.go

```diff
@@ -0,0 +1,122 @@
+package awsManager
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	clientpkg "github.com/openshift/aws-account-shredder/pkg/aws"
+	"os"
+	"strings"
+)
+
+// function to list all the EC2 instances
+// not removing this function right now , as it is easy to verify after deletion if it was actually deleted or not
+// keeping for future testing purpose ( easy to use )
+func ListEc2Instances(assumedRoleClient clientpkg.Client) {
```

Comment:
What's the functional difference between this and ListEc2InstancesForDeletion? Both do the same thing, not sure we need them both